app/templates/components/public/event-map.hbs

```diff
@@ -0,0 +1,9 @@
+<div class="eight wide column loc-map">
```

Comment:
Please use proper class names `loc-map` doesn't make any sense. You could call it `event-map`